PowerEditor/src/WinControls/OpenSaveFileDialog/FileDialog.cpp

```diff
@@ -474,15 +474,19 @@ BOOL APIENTRY FileDialog::run(HWND hWnd, UINT uMsg, WPARAM, LPARAM lParam)
 					{
 						// change to backslash, and insert trailing '\' to indicate directory
 						hFileDlg = ::GetParent(hWnd);
-						std::wstring _fnStr(fileName);

```

Comment:
As `_fnStr` is not member, preceding underscore is removed and it is changed to `filePath` which looks more appropriate as it is path.